style.css

```diff
@@ -96,9 +104,41 @@ body{
 }
 .category-list__list-item:hover{
     cursor: pointer;
-    background-color: #8080801c
+    background-color: #8080801c;
 }
+#more li{
+    border: 0.002rem solid #00000036;
+    border-top: none;
+    padding: 1.5rem;
+    display: none;
+}
+#dropdown {
+    position: absolute;
+    z-index: 1;
+    top: 14.2rem;
+    padding: 0rem;
+    display: none;
+    background: white;
+    border: 0.002rem solid #00000036;
+    border-top: none;
+    right: 10rem;
+    border-bottom-left-radius: 0.3rem;
+    border-bottom-right-radius: 0.3rem;
 
+}
+#more:hover #dropdown{
+    display: block;
+}
+#more:hover li{
```

Comment:
Line 129 and 132 can be written together like following:
`#more:hover #dropdown, #more:hover li {
 display: block;
}`